### PR TITLE
fix: several bugs, check body for detail

### PR DESCRIPTION
**Pull request by yqrashawn**
1. remove gh action checking fullnode err msg, it won't work. Better use https://github.com/returntocorp/semgrep later
2. fix `.post` in lock/unlock rpc method
3. fix wrong accounts auth in requestPermissions rpc method

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/356)

### CODE REVIEW

File: packages/rpcs/wallet_lock/index.js [hunk 1]
```diff
@@ -10,8 +10,12 @@ export const permissions = {
 export const main = async ({db: {setLocked, findApp}}) => {
   setLocked(true)
   const apps = findApp({g: {site: {post: 1}}})
-  apps.forEach(({site: {post}}) => {
+  apps.forEach(app => {
     try {
+      if (!app?.post) return
```
Comment (zctocm) [inline packages/rpcs/wallet_lock/index.js:15]: maybe `app?.site?.post`?

File: packages/rpcs/wallet_unlock/index.js [hunk 2]
```diff
@@ -58,23 +58,24 @@ export const main = async ({
   const apps = findApp({
     g: {currentNetwork: {eid: 1}, currentAccount: {eid: 1}, site: {post: 1}},
   })
-  apps.forEach(
-    ({
+  apps.forEach(app => {
+    if (!app?.post) return
```
Comment (zctocm) [inline packages/rpcs/wallet_unlock/index.js:62]: same as above

**Comment by yqrashawn** [comment]
above comments fixed @zctocm

**Comment by angelia-yuqi-personal** [comment]
Task linked: [PORTAL-1810 Fluent中Authorize Account 不工作，一个账号连dapp后，fluent中切换到另一账号（未授权过），点击Authorize进行授权，Fluent底部和dapp连接状态仍显示上个账号，dapp也不连接到新账号](https://app.clickup.com/t/3719212/PORTAL-1810)

**Review by zctocm — APPROVED** [review]

**Review by zctocm — APPROVED** [review]

**Pull request merged**